arrow-libs/fx/arrow-fx-coroutines/src/commonTest/kotlin/arrow/fx/coroutines/ResourceTest.kt

```diff
@@ -177,23 +183,26 @@ class ResourceTest : ArrowFxSpec(
       return Pair(promises.map { it.second }, res)
     }
 
-    "parZip - deep finalizers are called when final one blows" {
+    "parZip - deep finalizers are called when final one blows".config(enabled = !OS.isApple) {
```

Comment:
Are these still failing? They should pass for `iOS` :(

Reply:
Not in my machine, not in GitHub's :(

Reply:
Unfortunately I think this would need to stay. Anyway we're dropping them for Arrow 2.0...